src/main/java/minicraft/item/TileItem.java

```diff
@@ -73,7 +73,7 @@ protected TileItem(String name, Sprite sprite, int count, String model, List<Str
 		this.model = model.toUpperCase();
 		this.validTiles = new ArrayList<>();
 		for (String tile: validTiles)
-			 this.validTiles.add(tile.toUpperCase());
```

Comment:
This breaks all of the notifications in the file, and possibly more. Remove all changes to this file and I will merge.